docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>I'd always put UxDataArray, UxDataset, and Grid in "code" syntax using the backticks. For example, UxDataArrayI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>May you apply this to the sections below too? </p>